website/docs/r/search_index.html.markdown

```diff
@@ -0,0 +1,340 @@
+          }
+        }
+      },
+      "employees": {
+        "type": "string",
+        "analyzer": "lucene.standard"
+      }
+}
+EOF
+ name = "name_test"
+ searchAnalyzer = "lucene.standard"
+ analyzers {
+  name = "index_analyzer_test_name"
+  char_filters {
+   type = "mapping"
+   mappings = <<EOF
+   {"\\" : "/"}
+   EOF
+   }
+   tokenizer { 
+    type = "nGram"
+    min_gram = 2
+    max_gram = 5
+   }
+   token_filters = {
+    type = "length"
+    min = 20
+    max = 33
+   }
+  }
+}
+```
+
+## Argument Reference
+
+* `name` - (Required) The name of the search index you want to create.
+* `project_id` - (Required) The ID of the organization or project you want to create the search index within.
+* `cluster_name` - (Required) The name of the cluster where you want to create the search index within.
+
+
+* `analyzer` - [Analyzer](https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/#std-label-analyzers-ref) to use when creating the index. Defaults to [lucene.standard](https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/standard/#std-label-ref-standard-analyzer)
+
+* `analyzers` - [Custom analyzers](https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/custom/#std-label-custom-analyzers) to use in this index (this is an array of objects).
+
+* `collection_name` - (Required) Name of the collection the index is on.
+
+* `database` - (Required) Name of the database the collection is in.
+
+* `mappings_dynamic` - Indicates whether the index uses dynamic or static mapping. For dynamic mapping, set the value to `true`. For static mapping, specify the fields to index using `mappings_fields`
+
+* `mappings_fields` - attribute is required when `mappings_dynamic` is true. This field needs to be a JSON string in order to be decoded correctly.
+  ```hcl
+    mappings_fields = <<-EOF
+    {
+    "address": {
+    "type": "document",
+    "fields": {
+    "city": {
+    "type": "string",
+    "analyzer": "lucene.simple",
+    "ignoreAbove": 255
+    },
+    "state": {
+    "type": "string",
+    "analyzer": "lucene.english"
+    }
+    }
+    },
+    "company": {
+    "type": "string",
+    "analyzer": "lucene.whitespace",
+    "multi": {
+    "mySecondaryAnalyzer": {
+    "type": "string",
+    "analyzer": "lucene.french"
+    }
+    }
+    },
+    "employees": {
+    "type": "string",
+    "analyzer": "lucene.standard"
+    }
+    }
+  ```
+
+* `search_analyzer` - [Analyzer](https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/#std-label-analyzers-ref) to use when searching the index. Defaults to [lucene.standard](https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/standard/#std-label-ref-standard-analyzer)
+
+
+### Analyzers
+An Atlas Search analyzer prepares a set of documents to be indexed by performing a series of operations to transform, filter, and group sequences of characters. You can define a custom analyzer to suit your specific indexing needs.
```

Comment:
Let's add the URL to the main page - https://docs.atlas.mongodb.com/reference/atlas-search/analyzers/custom/.

Reply:
ok